docs/quickstart.md

```diff
@@ -182,4 +218,28 @@ $ kubectl delete providers/provider-oci-family
 
 > [!Warning]
 > Never delete a family provider before deleting its sub-providers. Deleting a family provider while sub-providers are still installed can lead to unexpected behavior and potential errors.
-  
\ No newline at end of file
+
+## Owner references conflict
+Crossplane’s package manager establishes an ownership chain between a family Provider (parent) and each sub‑provider (child) using metadata.ownerReferences on the child’s ProviderRevision. The owner reference encodes the parent Provider’s “package identity” as a name that the package manager expects to persist:
+
+```
+$ kubectl describe providerrevisions/provider-family-oci...
+
+  <TRUNCATED OUTPUT>
+  Owner References:
+    API Version:           pkg.crossplane.io/v1
+    Block Owner Deletion:  true
+    Controller:            true
+    Kind:                  Provider
+    Name:                  oracle-samples-provider-family-oci
+    UID:                   ...
+```
+
+What can go wrong
+- If the family Provider is deleted or reinstalled with a different metadata.name while any sub‑providers or their ProviderRevisions still exist, Crossplane will attempt to satisfy the outstanding owner reference by automatically pulling a Provider with the name encoded in the ownerReferences (from the same registry as the sub‑provider’s image).
+- This results in two family Providers trying to coexist (the one you applied and the one auto‑installed to satisfy the ownerRef), which leads to unhealthy/competing ProviderRevisions and confusing state during upgrades.
+
+Typical symptoms
+- An unexpected Provider with name - appears after you delete or rename your family Provider.
+- providerrevisions stuck in Unhealthy/Inactive with messages about conflicting ownership or multiple controlling owners.
+- Repeated reconcile loops pulling a family image you did not specify explicitly.
```

Comment:
We might want to add a bit about how to fix this if a user runs into this issue.